internal/provider/exemption_resource.go

```diff
@@ -0,0 +1,175 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-framework/path"
+	"github.com/hashicorp/terraform-plugin-framework/resource"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+
+	"github.com/hashicorp/terraform-provider-adaptive-metrics/internal/client"
+	"github.com/hashicorp/terraform-provider-adaptive-metrics/internal/model"
+)
+
+type exemptionResource struct {
+	client *client.Client
+}
+
+var (
+	_ resource.Resource                = &exemptionResource{}
+	_ resource.ResourceWithConfigure   = &exemptionResource{}
+	_ resource.ResourceWithImportState = &exemptionResource{}
+)
+
+func newExemptionResource() resource.Resource {
+	return &exemptionResource{}
+}
+
+func (e *exemptionResource) Configure(_ context.Context, req resource.ConfigureRequest, resp *resource.ConfigureResponse) {
+	if req.ProviderData == nil {
+		return
+	}
+
+	data, ok := req.ProviderData.(*resourceData)
+	if !ok {
+		resp.Diagnostics.AddError(
+			"Unexpected resource configure type",
+			fmt.Sprintf("Got %T. Please report this issue to the provider developers.", req.ProviderData),
+		)
+		return
+	}
+
+	e.client = data.client
+}
+
+func (e *exemptionResource) Metadata(_ context.Context, req resource.MetadataRequest, resp *resource.MetadataResponse) {
+	resp.TypeName = fmt.Sprintf("%s_exemption", req.ProviderTypeName)
+}
+
+func (e *exemptionResource) Schema(_ context.Context, _ resource.SchemaRequest, resp *resource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Required:    false,
+				Optional:    false,
+				Computed:    true,
```

Comment:
```suggestion
				Computed:    true,
```

You only need to set one of the three I believe